new-integration-tests/src/test/java/oracle/weblogic/kubernetes/extensions/ImageBuilders.java

```diff
@@ -142,6 +147,61 @@ public void close() {
       deleteImage(operatorImage);
     }
 
+    // delete images from OCIR, if necessary
+    if (REPO_NAME.contains("ocir.io")) {
+      String token = getOcirToken();
+      if (token != null) {
+        if (miiBasicImage != null) {
+          deleteImageOcir(token, miiBasicImage);
```

Comment:
The latest changes in develop branch seems to build more images in this class for ItDomainInImageWdt.java. Are we going to delete all those images?